src/dnx.coreclr.unix/dnx.coreclr.cpp

```diff
@@ -244,10 +244,10 @@ int InvokeDelegate(host_main_fn host_main, int argc, const char** argv, const bo
 
 std::string get_os_version()
 {
-    std::vector<std::string> qualifiers { "DISTRIB_ID=", "DISTRIB_RELEASE=" };
+    std::vector<std::string> qualifiers { "ID=", "VERSION_ID=" };
```

Comment:
You are right: _Variable assignment values must be enclosed in double or single quotes if they include spaces, semicolons or other special characters outside of A-Z, a-z, 0-9_